Scrapper-Service/database/mdb.py

```diff
@@ -63,8 +67,17 @@ def put(self , conference_data):
         else:
             _id = conference_data._id
             try:
-                res = self.collection.update_one( {'_id':_id}  ,conference_data.get_query(), upsert = True)
+                res = self.collection.update_one( {'_id':_id,'deadline':{'$gte':conference_data.querydata['deadline']}}  ,{'$set' :conference_data.query_dict()} , upsert = True)
```

Comment:
* based on res.modified.count you can now call some call backs in. 
* All less looks fine from my end.